dictionaries/software-terms/cspell-corrections.yaml

```diff
@@ -53,3 +53,23 @@ dictionaryDefinitions:
       - transfor->transfer, transform
       - verticies->vertices
       - warnining->warning
+
+      - comited->committed
+      - commited->committed
+      - comitted->committed
+
+      - commiter->committer
+      - comiter->committer
+      - comitter->committer
+
+      - commiters->committers
+      - comiters->committers
+      - comitters->committers
+
+      - comitting->committing
+      - comiting->committing
+      - commiting->committing
+
+      - commitable->committable
+      - comitable->committable
+      - comittable->committable
```

Comment:
I'm unsure this file is automatically sorted, should I ?

Reply:
This file is not automatically sorted. I haven't made a tool for that yet (might be a while).

Reply:
I opened #4442

- #4442